roles/edpm_kernel/vars/main.yml

```diff
@@ -20,51 +20,87 @@ edpm_kernel_modules:
   nf_conntrack: {}
 
 edpm_kernel_sysctl_settings:
-  net.ipv4.tcp_keepalive_intvl:
+  fs.inotify.max_user_instances:
+    value: 1024
+  fs.suid_dumpable:
+    value: 0
+  kernel.dmesg_restrict:
     value: 1
-  net.ipv4.tcp_keepalive_probes:
-    value: 5
-  net.ipv4.tcp_keepalive_time:
-    value: 5
-  net.ipv4.conf.default.send_redirects:
+  kernel.pid_max:
+    value: 1048576
+  net.bridge.bridge-nf-call-arptables:
+    value: 1
+  net.bridge.bridge-nf-call-ip6tables:
+    value: 1
+  net.bridge.bridge-nf-call-iptables:
+    value: 1
+  net.core.netdev_max_backlog:
+    value: 10000
+  net.ipv4.conf.all.arp_accept:
+    value: 1
+  net.ipv4.conf.all.arp_notify:
+    value: 1
+  net.ipv4.conf.all.log_martians:
+    value: 1
+  net.ipv4.conf.all.rp_filter:
+    value: 1
+  net.ipv4.conf.all.secure_redirects:
     value: 0
   net.ipv4.conf.all.send_redirects:
     value: 0
-  net.ipv4.conf.all.arp_accept:
-    value: 1
   net.ipv4.conf.default.accept_redirects:
     value: 0
+  net.ipv4.conf.default.log_martians:
+    value: 1
   net.ipv4.conf.default.secure_redirects:
     value: 0
-  net.ipv4.conf.all.secure_redirects:
+  net.ipv4.conf.default.send_redirects:
     value: 0
-  net.ipv4.conf.default.log_martians:
+  net.ipv4.ip_forward:
     value: 1
-  net.ipv4.conf.all.log_martians:
+  net.ipv4.ip_local_reserved_ports:
+    value: "35357,49000-49001"
+  net.ipv4.ip_nonlocal_bind:
     value: 1
-  net.nf_conntrack_max:
-    value: 500000
-  net.netfilter.nf_conntrack_max:
-    value: 500000
+  net.ipv4.neigh.default.gc_thresh1:
+    value: 1024
+  net.ipv4.neigh.default.gc_thresh2:
+    value: 2048
+  net.ipv4.neigh.default.gc_thresh3:
+    value: 4096
+  net.ipv4.tcp_keepalive_intvl:
+    value: 1
+  net.ipv4.tcp_keepalive_probes:
+    value: 5
+  net.ipv4.tcp_keepalive_time:
+    value: 5
   net.ipv6.conf.all.accept_ra:
     value: 0
-  net.ipv6.conf.default.accept_ra:
+  net.ipv6.conf.all.accept_redirects:
     value: 0
   net.ipv6.conf.all.autoconf:
     value: 0
-  net.ipv6.conf.default.autoconf:
+  net.ipv6.conf.all.disable_ipv6:
+    value: 0
+  net.ipv6.conf.all.forwarding:
+    value: 0
+  net.ipv6.conf.all.ndisc_notify:
+    value: 1
+  net.ipv6.conf.default.accept_ra:
     value: 0
   net.ipv6.conf.default.accept_redirects:
     value: 0
-  net.ipv6.conf.all.accept_redirects:
+  net.ipv6.conf.default.autoconf:
     value: 0
-  net.ipv4.conf.all.arp_notify:
-    value: 1
-  net.ipv6.conf.all.ndisc_notify:
+  net.ipv6.conf.default.disable_ipv6:
+    value: 0
+  net.ipv6.conf.lo.disable_ipv6:
+    value: 0
+  net.ipv6.ip_nonlocal_bind:
     value: 1
-  net.core.netdev_max_backlog:
-    value: 10000
-  kernel.dmesg_restrict:
+  net.netfilter.nf_conntrack_max:
+    value: 500000
+  net.nf_conntrack_max:
+    value: 500000
+  vm.unprivileged_userfaultfd:
```

Comment:
as a follow up you can remove this for the nova role.

Reply:
we can do in a single swath, by https://github.com/openstack-k8s-operators/edpm-ansible/pull/94, as we address all the comments there